sentences/lb/homeassistant_HassTurnOn.yaml

```diff
@@ -8,16 +8,6 @@ intents:
           - aktivéier <name>
           - <maach> <area> <name> <op>
           - <maach> <name> <area> <op>
-      - sentences:
-          - <maach> <all_window> <op>
-        response: cover
-        slots:
-          domain: cover
-          device_class:
-            - window
-            - blind
-            - curtain
-            - shutter
       - sentences:
```

Comment:
Lines 11-13: Can remain here if there a) is not going to be a fileset for doors, b) doors (e.g. garage door) not considered as covers. Else, move to new fileset.